bin/export-import-data

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/perl
+
+use v5.14;
+use warnings;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname(File::Spec->rel2abs($0));
+    require "$d/../setenv.pl";
+}
+
+use FixMyStreet::DB;
+use Getopt::Long::Descriptive;
+use JSON::MaybeXS;
+use Path::Tiny;
+
+my ($opt, $usage) = describe_options(
+    '%c',
+    [ 'name=s', "Name of body" ],
+    [ 'import=s', "File to import" ],
+    [ 'commit', "Actually commit changes to the database" ],
+    [ 'help', "print usage message and exit", { shortcircuit => 1 } ],
+);
+print($usage->text), exit if $opt->help;
+$usage->die unless $opt->name;
+die "Please specify a file to import\n" if $opt->import && ! -e $opt->import || -d $opt->import;
+
+my $J = JSON::MaybeXS->new(utf8 => 1, pretty => 1);
```

Comment:
Also need to add something like

`use open ':std', ':encoding(UTF-8)';`

so that the output is UTF-8 encoded otherwise you get an error on import.

Reply:
Hmm, I think that'll double decode it then, doesn't stop the wide character warning anyway. I changed the slurp_utf8 to just slurp, as JSON is doing the utf8 decoding for us here so don't need to do it twice.